v0/pipeline/steps/custom/output-alias.yaml

```diff
@@ -0,0 +1,18 @@
+title: OutputAlias
+type: object
+required:
+  - key
+  - scope
+properties:
+  key:
+    type: string
+    minLength: 1
+  scope:
+    type: string
+    enum:
+      - Pipeline
+      - Stage
+      - StepGroup
+  description:
+    desc: This is the description for OutputAlias
```

Comment:
Please describe what this class is used for

Reply:
done